.github/workflows/release.yml

```diff
@@ -130,3 +136,10 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
             INFURA_KEY=${{ secrets.INFURA_KEY }}
+
+      - name: Merge changes back into develop for latest release (master)
+        if: ${{ env.TAG == 'latest' }}
+        run: |
+          git checkout develop
+          git merge master
+          git push origin develop
```

Comment:
I worry what will happen if someone commits to `develop` while a release is running. Will the merge cause things to be out of order. Could we end up with a timeline like this:

```
chore(release): publish v7.0.4
feat: a fix to land in v7.1.0
fix: v7.0.3 was broken, this fixes it
```

Would it be best to back-date our `chore(release):` commit to the same timestamp as the commit (tip of `master`) that initiated the release? I just don't know enough about git :thinking:

Reply:
I think it would fail when pushing to origin/develop @davidmurdoch. The commits are on local refs and resolved by either git fetch, git push, both of which would fail if the tips don't match.

Reply:
@davidmurdoch I also intend to restrict pushing to `develop` by anyone other than TrufBot. For now we have few enough people that we'll just need to coordinate during releases.